.github/workflows/build.yml

```diff
@@ -39,3 +39,26 @@ jobs:
         - name: Test
           shell: bash
           run: cd build && ctest --output-on-failure
+
+  bz-build:
```

Comment:
Name this "bazel-build" (no need for abbreviation here).

And can you rename the other one "cmake-build"?

Reply:
Done